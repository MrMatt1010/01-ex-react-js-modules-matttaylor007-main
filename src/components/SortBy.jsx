const SortBy = () => {
  return (
    <div className="SortBy">
      <label htmlFor="sort">Sort by</label>
      <select id="sort" defaultValue="latest">
        <option value="latest">Latest</option>
        <option value="low">Price: Low</option>
        <option value="high">Price: High</option>
      </select>
    </div>
  );
};

export default SortBy;
