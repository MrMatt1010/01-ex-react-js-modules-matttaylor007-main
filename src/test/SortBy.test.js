import { render, cleanup } from "@testing-library/react";
import SortBy from "../components/SortBy";

afterEach(cleanup);

test("renders the SortBy component as expected", () => {
  const { asFragment } = render(<SortBy />);
  expect(asFragment()).toMatchSnapshot();
});
