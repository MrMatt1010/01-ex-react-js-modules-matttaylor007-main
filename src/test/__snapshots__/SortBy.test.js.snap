// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the SortBy component as expected 1`] = `
<DocumentFragment>
  <div
    class="SortBy"
  >
    <label
      for="sort"
    >
      Sort by
    </label>
    <select
      id="sort"
    >
      <option
        selected=""
        value="latest"
      >
        Latest
      </option>
      <option
        value="low"
      >
        Price: Low
      </option>
      <option
        value="high"
      >
        Price: High
      </option>
    </select>
  </div>
</DocumentFragment>
`;
